kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -642,4 +646,55 @@ private void configurationEnforcementDisable() throws DBusException {
         logger.debug("Network configuration enforcement set to {} (Expected: false)",
                 this.configurationEnforcementHandlerIsArmed);
     }
+
+    public List<Location> getAvailableMMLocations() {
+        List<Location> availableLocations = new ArrayList<>();
+
+        this.getEnabledModemsPaths().stream().forEach(modemPath -> {
```

Comment:
I think that you can use the `forEach` method directly to the List, without using `stream()`